packages/lab/stories/static-list/static-list.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import { Button, Divider, Label, StackLayout, Text } from "@salt-ds/core";
+import { NoteIcon, NotificationIcon } from "@salt-ds/icons";
+import { StaticList, StaticListItem, type StaticListProps } from "@salt-ds/lab";
+import type { Meta, StoryFn } from "@storybook/react";
+import { useState } from "react";
+
+const containerStyle = {
+  display: "flex",
+  justifyContent: "center",
+  width: "calc(100vw - 2em)",
+};
+
+export default {
+  title: "Lab/Static List",
+  component: StaticList,
+} as Meta<typeof StaticList>;
+
+const ListItemWithIcons = () => (
+  <StaticListItem
+    style={{ padding: "var(--salt-spacing-50) var(--salt-spacing-100)" }}
+  >
+    <NotificationIcon />
+    <StackLayout
+      gap={0.5}
+      style={{ padding: "var(--salt-spacing-75) 0", width: "100%" }}
+    >
+      <Text color="inherit">Item label</Text>
+      <Label color="secondary">Secondary label</Label>
```

Comment:
Label is wrong semantically although <Text styleAs="label" /> should probably be used instead.

```suggestion
      <Text styleAs="label" color="secondary">Secondary label</Label>
```